config.js

```diff
@@ -1,8 +1,6 @@
-var dotenv = require('dotenv');
+// Do not allow the application to run if env vars not set in .env file
+require('dotenv-safe').load();
 
-dotenv.config({path: '.env'});
-
-var twimlApp = require('./util/twimlApp');
 var cfg = {};
```

Comment:
I know you're not the one who wrote this, but why not to use an object literal? IMHO it is more readable. Besides that there is no real gain on abbreviate `config` to `cfg`.

```javascript
module.exports = {
  secret: process.env.APP_SECRET || 'keyboard cat',
  accountSid: process.env.TWILIO_ACCOUNT_SID
  // ...
};
```